awx/ui_next/src/screens/Project/Project.jsx

```diff
@@ -121,6 +120,16 @@ class Project extends Component {
     const canToggleNotifications = isNotifAdmin;
 
     const tabsArray = [
+      {
```

Comment:
Why are we adding this tab here?  No other resource has this tab unless it is a nested resource like Inventory Sources

Reply:
we are removing the back button and moving to this pattern.  i need to add this to the rest of the views

Reply:
this is good to know because I'm working on a details view right now